src/main/java/io/bf2/kafka/authorizer/AuditLoggingController.java

```diff
@@ -61,6 +61,22 @@ public class AuditLoggingController implements Configurable, Closeable {
     private Cache<CacheKey, CacheEntry> loggingEventCache;
     private Set<ApiKeys> suppressApis = Collections.emptySet();
 
+    /**
+     * Primary constructor
+     */
+    public AuditLoggingController() {
+        this("AuditEvents");
+    }
+
+    /**
+     * primarily for tests to increase isolation.
+     * Identifies which logger to direct audit events too.
+     * @param auditLoggerKey the name for
```

Comment:
1. `Identifies which logger to direct audit events too.` -> Should we remove the `too` in the end? I don't know what it means here
2. @param auditLoggerKey the name for [WHAT?]

Reply:
1. Should be `to` not `too`. Trying to convey the direction of travel of the audit events.
2. Good catch.